### DISCUSSION TITLE
Equivalent of changesets/action for circleci ?

### DISCUSSION THREAD

**Discussion by NathanVss** [changesets/changesets#1067] [Q&A] [answered]
Hi there, I love the package, so well done 🙏

I would like to know if someone already used CircleCI to automate the creation of the versioning PR that is automatically made by [https://github.com/changesets/changesets/blob/main/docs/automating-changesets.md ? ](https://github.com/changesets/action) ? I am really interest in it, because I can't use Github Actions for business reasons as we already have an CircleCI setup 😕

Thank you in advance.

**Comment by Andarist** [comment, marked as answer]
At the moment there is nothing builtin to accomplish this - but you could copy-paste `changesets/action` and introduce the required changes. Out [`@changesets/release-utils`](https://github.com/changesets/changesets/blob/8447c677a37d1d23858059e0ce2b5cd98f4c24ae/packages/release-utils/package.json) might also make it a little bit easier. I think that there is potential in making it even easier but personally I don't have time to work on this right now.

**Reply by NathanVss** [reply]
Thank you for your reply @Andarist, we'll see what we can do on our side 🙏
